packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx

```diff
@@ -52,7 +52,7 @@ const WorkspaceHeader: React.FC<{}> = () => {
       (isResourceRoute && currentResource) ||
       (isResourceRoute && currentProjectConfiguration)
     )
-      return currentResource?.name || currentProjectConfiguration?.name;
+      return currentResource?.resourceType ===  models.EnumResourceType.Service ?  currentResource?.name : 'Project Configuration'
```

Comment:
use a const instead of hard-coded value and use this const in other pages 

export const PROJECT_CONFIGURATION_RESOURCE_NAME = "Project Configuration";

Reply:
Done